doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -84,7 +84,8 @@ Developing with nRF52 Series
 Developing with Front-End Modules
 =================================
 
-|no_changes_yet_note|
+* When the nRF21540EK shield is used, explicit use of ``-DSHIELD=nrf21540ek_fwd`` for boards with qualifiers ``nrf5340/cpuapp`` is no more necessary and deprecated.
+  The build system uses an appropriate overlay file for each core relying only on the ``-DSHIELD=nrf21540ek`` parameter.
```

Comment:
```suggestion
* Removed the explicit use of ``-DSHIELD=nrf21540ek_fwd`` for boards with ``nrf5340/cpuapp`` qualifiers when the nRF21540 EK shield is used.
  The build system uses an appropriate overlay file for each core, relying only on the ``-DSHIELD=nrf21540ek`` parameter.
```
Aren't we removing (not deprecating) this parameter and ``-DSHIELD=nrf21540ek`` is used, instead?

Reply:
The file `boards/shields/nrf21540ek/nrf21540ek_fwd.overlay` still exists and has the same result as previously (on app core build system will include pin forwarding overlay). Technically one could still use the `-D<app_core_image>_SHIELD=nrf21540ek_fwd` . So the old way is deprecated not totally removed. This is to give users time to adjust.

Reply:
Then, we could rephrase as above.

Reply:
rephrased